examples/common/chip-app-server/Server.cpp

```diff
@@ -195,21 +108,28 @@ void InitServer()
 {
     CHIP_ERROR err = CHIP_NO_ERROR;
     Optional<Transport::PeerAddress> peer(Transport::Type::kUndefined);
+#ifndef BYPASS_RENDEZVOUS
```

Comment:
Could we place this together with the code below (eventually within {} if the sucessorexit forces parameter declaration)?

it would make it easier to read if code is localized. We can also split it into a separate function.
It looks like pinCode is also only needed for rendezvous.